libs/components/popovers/src/lib/modules/popover/popover.directive.ts

```diff
@@ -19,25 +19,40 @@ export class SkyPopoverDirective implements OnInit, OnDestroy {
    * @required
    */
   @Input()
-  public skyPopover: SkyPopoverComponent;
+  public set skyPopover(value: unknown) {
```

Comment:
Now that you're removing the code example where this is set in the component's class, I'd revert this back to a simple public property of type `SkyPopoverComponent` with no getter/setter.